include/Core/Base/Mixin/Renderer.php

```diff
@@ -111,4 +111,61 @@ protected function render_user_role_dropdown() {
 	protected function render_post_type_dropdown() {
 		bd_render_post_type_dropdown( $this->field_slug );
 	}
+
```

Comment:
@rajanit2000, 

Also, verify if you're sticking to standards in this file too.